R/metadata_orr_scrape.R

```diff
@@ -0,0 +1,187 @@
+#' Download the cover sheet from a published ORR table
```

Comment:
Doesn't look like you've run roxygen2::roxygenise() to pick up this new documentation